src/components/molecules/Visualizer/Engine/index.tsx

```diff
@@ -30,6 +30,23 @@ export type SceneProperty = {
     bgcolor?: string;
     ion?: string;
   };
+  cameraLimiter?: {
+    id: string;
+    cameraLimitterEnabled?: boolean;
+    cameraLimitterShowHelper?: boolean;
+    cameraLimitterTargetArea: {
+      altitude: number;
+      fov: number;
+      heading: number;
+      height: number;
+      lat: number;
+      lng: number;
+      pitch: number;
+      roll: number;
+    };
```

Comment:
Camera type is imported at https://github.com/reearth/reearth-web/blob/main/src/components/molecules/Visualizer/Engine/index.tsx#L12

```suggestion
    cameraLimitterTargetArea?: Camera;
```